website/frontend/templates/docs/build-windows.html

```diff
@@ -6,113 +6,192 @@
+        then a separate branch 1.4.x is available to run and this uses Python 2 and PqQt4. 
+        Instructions for building each environment are provided below.
+      </p>
+      
+      <h2 id="source">Downloading Picard source</h2>
+      
+      <p>
+        If you are planning to develop and submit changes to Picard source
+        then you should install <a rel="nofollow" target="_blank" href="https://desktop.github.com/">GitHub Desktop</a> 
+        or an alternative Git tool of your choice and use it to create your own local Git clone of Picard.
+      </p>
 
-      <h3 id="python">Python 2.7.x</h3>
+      <p>
+        But if you are planning only to run from source (rather than use the Picard installer / executable), 
+        then you can download and unzip the current master branch from 
+        <a rel="nofollow" href="https://github.com/metabrainz/picard/archive/master.zip">https://github.com/metabrainz/picard/archive/master.zip</a> 
+        or the 1.4.x branch from <a rel="nofollow" href="https://github.com/metabrainz/picard/archive/1.4.x.zip">https://github.com/metabrainz/picard/archive/1.4.x.zip</a>.
+      </p>
 
-      <ul>
-        <li><a rel="nofollow" href="http://python.org/download/">http://python.org/download/</a>
-        </li>
-        <li>Download and install "Python 2.7.x Windows Installer"</li>
-        <li>Add C:\Python27 to&#160;%PATH%</li>
-      </ul>
+      <h2 id="python_picard2">Python for Picard version 2</h2>
 
-      <h3 id="pyqt">PyQt 4.10.x</h3>
+      <h3 id="python35">Python 3.5 or newer</h3>
 
       <ul>
-        <li> <a rel="nofollow" href="http://www.riverbankcomputing.co.uk/software/pyqt/download">http://www.riverbankcomputing.co.uk/software/pyqt/download</a>
+        <li>
+          Download and install Python 3 for Windows from 
+          <a rel="nofollow" target="_blank" href="http://python.org/downloads/">http://python.org/downloads/</a>. 
+          The latest version is recommended, but if you already have Python 3 installed
+          then the minimum version is 3.5.
+        </li>
+        <li>
+          Python should add itself to your %PATH% environment variable, 
+          but if not add both the Python install directory and its scripts subdirectory to %PATH%
         </li>
-        <li>Download and install "PyQt4-4.10.x-gpl-Py2.7-Qt4.8.x-x32.exe"</li>
       </ul>
 
-      <h3 id="libdiscid">libdiscid</h3>
+      <h3 id="python3_packages">Python 3 packages</h3>
+      
+      <p>
+         You need need the following additional python packages to run Picard from source:
+      </p>
 
       <ul>
-        <li> <a rel="nofollow" href="http://musicbrainz.org/doc/libdiscid">http://musicbrainz.org/doc/libdiscid</a>
-        </li>
-        <li>Download "libdiscid-0.6.1-win32.zip" and put "discid.dll" in your Windows\System32 folder</li>
+        <li>PyQt 5.7.1 or later</li>
+        <li>Mutagen 1.37 or later</li>
+        <li>discid</li>
```

Comment:
pip3 should download libdiscid automatically as a dependency.

However I did this on my PC which already supported Picard 1 and I haven't tried this on a clean PC, so I cannot be completely certain that this will work from scratch.